intake_bluesky/core.py

```diff
@@ -676,9 +677,10 @@ def __init__(self,
         self._ds = None  # set by _open_dataset below
         self.include = include
         self.exclude = exclude
-        super().__init__(
-            metadata=metadata
-        )
+        super().__init__(metadata=metadata)
+
+    def filled_event_pages(self):
+        yield from self._filler.fill(self)
```

Comment:
What is this intended for? Seems like this is passing a `BlueskyEventStream` instance to a method of `event_model.Filler` that doesn't exist.

Reply:
removed, this was left over from previous changes that i made